core/src/main/java/org/pentaho/platform/scheduler2/ws/DefaultSchedulerService.java

```diff
@@ -114,24 +117,26 @@ private void updateJob( String jobId, Map<String, ParamValue> jobParams, JobTrig
    * {@inheritDoc}
    */
   public Job[] getJobs() throws SchedulerException {
+
     IScheduler scheduler = PentahoSystem.get( IScheduler.class, "IScheduler2", null ); //$NON-NLS-1$
     IPentahoSession session = PentahoSessionHolder.getSession();
     String principalName = session.getName();
-    Boolean canAdminister = PentahoSystem.get( IAuthorizationPolicy.class ).isAllowed( ADMIN_PERM );
+    boolean canAdminister = POLICY.isAllowed( ADMIN_PERM );
+    boolean canExecuteScheduler = canExecuteScheduler();
 
     return scheduler.getJobs( job -> {
-      if ( canAdminister ) {
+      if ( canAdminister || canExecuteScheduler ) {
```

Comment:
please add a `testGetJobsSchedulePowerUser` test here  [DefaultSchedulerServiceTest.java](https://github.com/pentaho/pentaho-scheduler-plugin/blob/master/core/src/test/java/org/pentaho/platform/scheduler2/ws/test/DefaultSchedulerServiceTest.java#L105)

Reply:
@afrjorge tests are ignored in this file. check this [comment](https://github.com/pentaho/pentaho-scheduler-plugin/pull/170#issuecomment-2100814306)